cloudsql/sqlserver/database-sql/templates/index.html

```diff
@@ -60,7 +60,7 @@ <h4 class="header center">Recent Votes</h4>
     <ul class="container collection center">
         {{ range .RecentVotes }}
             <li class="collection-item avatar">
-                {{ if eq .Candidate "TABS" }}
+                {{ if eq  .Candidate "TABS  " }}
```

Comment:
Intentional change?

Reply:
Unfortunately, yes. The SQLServer column used here mandates _exact_ character length (as opposed to a maximum length), so it pads data with white space 😭